packages/@aws-cdk/aws-dynamodb/test/dynamodb.test.ts

```diff
@@ -719,6 +722,71 @@ test('if an encryption key is included, encrypt/decrypt permissions are added to
   });
 });
 
+test('when specifying STANDARD_INFREQUENT_ACCESS table class', () => {
+  const stack = new Stack();
+  new Table(stack, CONSTRUCT_NAME, {
+    tableName: TABLE_NAME,
```

Comment:
Any reason for specifying this here? Pretty sure this is optional.

Reply:
tableName is indeed optional